### ISSUE TITLE
Unexpected package creation during export data loading

### ISSUE THREAD

**Issue by ankitpatial** [ent/ent#2826]
With go1.19 ent generate give the following error
_Unexpected package creation during export data loading_
it seems to be related to entgo.io/contrib

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/ent/ent/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯
ent schema with entgo.io/contrib init will cause it to fail the command > go generate ./ent/  

## Expected Behavior 🤔
command > go generate ./ent/ 
must execure without error

## Steps to Reproduce 🕹
Try this
```go
package schema

import (
	"entgo.io/contrib/entgql"
	"entgo.io/ent"
	"entgo.io/ent/schema/field"
)

// User holds the schema definition for the User entity.
type User struct {
	ent.Schema
}

// Fields of the User.
func (User) Fields() []ent.Field {
	return []ent.Field{
		field.String("username").
			Unique().
			NotEmpty().
			MaxLen(50),
		field.String("email").
			NotEmpty().
			Unique().
			Annotations(
				entgql.OrderField("EMAIL"),
			),
	}
}

```

Steps:

1. init a entity User 
2. Copy above code to it
3. Try executing > go generate ./ent/

## Your Environment 🌎

| Tech        | Version |
| ----------- | ------- |
| Go          | 1.19  |
| Ent         | 0.11.1   |
| Database    | MySQL   |
| Driver      | https://github.com/go-sql-driver/mysql        |

**Comment by ankitpatial** [comment]
Quick update:

I tried coping source code of contrib/entgql package to my project local folder made chnages for path and removed most of things but just the just a annotations.go file. I furthre stripped it and having least code with only  2 imports "encoding/json" & "entgo.io/ent/entc/gen"

the moment I remove "entgo.io/ent/entc/gen"  issue goes away & comand >  go genererate ./ent 
works.

**Comment by ankitpatial** [comment]
I guess that might be related to package "golang.org/x/tools/go/packages" used in /entc/entc.go file

**Comment by ankitpatial** [comment]
OK, seems like solution may be just to upgrade the package "golang.org/x/tools"

**Comment by a8m** [comment]
Fixed with #2832 

Thanks for reporting this 🙏
